site/content/reference/01-commands.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Command Reference"
+weight: 10
+---
+
+## eksctl commands
+
+```
+a CLI for Amazon EKS
+
+Usage: eksctl [command] [flags]
+
+Commands:
+  eksctl completion  Generates shell completion scripts
+  eksctl create      Create resource(s)
+  eksctl delete      Delete resource(s)
+  eksctl drain       drain resources(s)
+  eksctl get         Get resource(s)
+  eksctl help        Help about any command
+  eksctl scale       Scale resources(s)
+  eksctl update      Update resource(s)
+  eksctl utils       Various utils
+  eksctl version     Output the version of eksctl
+
+Use 'eksctl [command] --help' for more information about a command.
+```
+
+`Usage: eksctl [command] [flags]`
+
+##### common flags<a name="common-flags"></a>
```

Comment:
I would not use `h4` or `h5` since they are small enough to look confusing in the context of our base font size.

Reply:
OK - let's connect Monday morning and figure out how to amend the tags.  I want it to look good as rendered, but it should remain a structured doc like an outline format somehow.